src/components/elements/headers.tsx

```diff
@@ -7,31 +7,31 @@ const headingLinkClasses = "[&_a]:hocus:underline"
 
 export const H1 = ({children, className, ...props}: Props) => {
   return (
-    <h1 className={twMerge("type-5 font-medium", className)} {...props}>
+    <h1 className={twMerge("type-4 font-medium", className)} {...props}>
```

Comment:
@pookmish I realized while triple checking things that the headings defined for the [WYSIWYG designs](https://www.figma.com/design/KSANrYJp8TW671KCamn3Dg/SU-Press-Redesign?node-id=3529-67884&t=mgCL1qSCqKAsMQ2k-1) differ from the standard/summer headings. I did a mass fixup again and clicked through the site and reviewed the changes. We should be all set now but I might roll up some additional changes when I work on [SUP-240 digital project page ](https://stanfordits.atlassian.net/browse/SUP-240)